src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -453,7 +452,6 @@ public void ValidateBlock(RuleContext context, bool skipRules = false)
                 }
 
                 // Check the block header is correct.
-                this.Validator.CheckBlockHeader(context);
```

Comment:
Where did this rule move to? If its not valid anymore, then perhaps remove the comment as well ;)

Reply:
Oh, is this new `CalculateStakeRule` rule? Then just remove the comment 👍

Reply:
Done